cmd/limactl/shell.go

```diff
@@ -126,9 +130,24 @@ func shellAction(cmd *cobra.Command, args []string) error {
 		)
 	}
 
-	arg0, err := exec.LookPath("ssh")
-	if err != nil {
-		return err
+	var arg0 string
+	var arg0Args []string
+
+	if sshShell := os.Getenv(envShellSSH); sshShell != "" {
+		sshShellFields := strings.Fields(sshShell)
```

Comment:
I had simple use cases in mind, but you're raising an excellent point.

The feedback I received in the linked issue seems to indicate that the implementation is going to change to pass arguments to `$SHELL -c` instead of overriding the command in Lima. I'll leave this open for a bit in case somebody wants to chime in.

Reply:
Done in [b6645274..5ca81769](https://github.com/lima-vm/lima/compare/b66452748ceae0d89c7da0acdfeb5c1db52098b9..5ca81769c4da3a72edb29156ceea99464a745758).

PTAL